ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/_cq_dialog/.content.xml

```diff
@@ -279,6 +279,17 @@
                                                     </cui>
                                                 </uiSettings>
                                             </thankYouMessage>
+                                            <dorAbsenceMessage
+                                            granite:class="warning-icon"
+                                            granite:id="dorWarningIcon"
+                                            jcr:primaryType="nt:unstructured"
+                                            sling:resourceType="/libs/fd/af/components/dorAbsenceBanner"
```

Comment:
is it in public area. we cannot use anything in core components that is internal.

Reply:
Raised a pr for adding the FinalArea property for the components. https://git.corp.adobe.com/livecycle/cq-guides/pull/5456/files